README.md

```diff
@@ -1 +1,129 @@
-# Awesome-Books
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Awesome books</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Awesome Books\] ](#-awesome-books-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+A simple Book storing web app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+1. Linters
+2. HTML5
+3. JS (ES6 synthax)
+4. Luxon
+
+### Key Features <a name="key-features"></a>
+
+
```

Comment:
- [ ] It would be better if we add the points under both the future and key features, currently we only have the heading without the bullet points as seen in the screenshot below
![Screenshot from 2023-07-10 18-38-29](https://github.com/GhostEsso/Awesome-Books/assets/46388113/9d09ce78-04f8-460b-9595-11e584184fe1)
![Screenshot from 2023-07-10 18-38-19](https://github.com/GhostEsso/Awesome-Books/assets/46388113/62d314cd-2af4-47aa-8a2e-322492aff1e9)